src/single-bulb-learning-module.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Template file used to render a single learning module.
+ *
+ * @package bu-learning-blocks
+ */
+
+get_header();
+?>
```

Comment:
Closing and opening php tags here also may not be necessary?

Reply:
Done